content/apps/meteor.md

```diff
@@ -0,0 +1,145 @@
+---
+menu:
+  main:
+    parent: apps
+title: Deploying Meteor
+weight: 10
+---
+
+# About Meteor
+
+[Meteor](https://www.meteor.com/) is a Javascript framework that makes it very easy to synchronize data between the client and server. Moreover it's also very easy to share code between client and server.
+
+Meteor includes nearly everything you need to get a web application up and running:
+
+- Javascript server framework
+- Javascript client framework
+- MongoDB for persistence
+- Asset pipeline (automagically include and minify CSS, JS, etc)
+
+# Hello world
+
+## Install Meteor locally
+
+See https://www.meteor.com/install. On OSX or Linux you can just run this in the terminal:
+
+```
+$ curl https://install.meteor.com/ | sh
+```
+
+## Create a boilerplate app
+
+```
+$ meteor create hello
+$ cd hello
+$ meteor
+# this starts the Meteor server
+# view the local site in a browser to ensure everything works
+# press ctrl-c to exit the server when you are done
+```
+
+## Adding the manifest.yml
+
+Create at the root of your application a file called `manifest.yml` and add:
+
+```yaml
+---
+applications:
+- name: hello-meteor
+  memory: 128M
+  instances: 1
+  domain: 18f.gov
+env:
+  FORCE_HTTPS: true
+```
+
+You should to customize the application name.
+
+## Deploy
+
+Push using the [`cf-meteor-buildpack`](https://github.com/cloudfoundry-community/cf-meteor-buildpack).
+
+```
+$ cf push hello-meteor -b https://github.com/csterwa/cf-meteor-buildpack.git
+```
```

Comment:
How about adding this to the manifest?

https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#buildpack